### DISCUSSION TITLE
Hacktoberfest! ✨

### DISCUSSION THREAD

**Discussion by jbolda** [tauri-apps/tauri#1032] [General]
[Hacktoberfest](https://hacktoberfest.digitalocean.com/) has arrived! We have a few places that might be interesting for newcomers and seasoned folk alike. I will keep this updated as the month progresses.

You can find our general roadmap on the docs website, https://tauri.studio/#roadmap, at the bottom of the page. It may need to have a few items pushed back as the core team has been pretty preoccupied with those pesky jobs and life in general, but we would love to have your help keeping things moving. We are also looking for contributors interested in working more long term with Tauri. This has taken the form of [code teams](https://github.com/tauri-apps/governance-and-guidance/pull/11).

Make sure to [join our Discord server](https://discord.gg/SpmNs4S)! Let us know that you are looking to contribute for Hacktoberfest and we will help get you started in the right direction. We can also answer any other questions you may have about Tauri.

on the js side:
- [ ] probably the big thing is to work on getting yarn create tauri-app working (we have the package, just need to shift things around). [ contacts: @nklayman @jbolda ] ✨✨
- [ ] any starter projects for the big frameworks (react is the only one so far) ✨
- [ ] any open issues ✨?

on the rust side:
- [ ] finish core CLI rewrite in Rust [ PR #851, contacts: @lucasfernog @jbolda ] ✨✨✨
- [ ] updater [ PR #643, contacts: @jbolda ] ✨✨✨✨
- [ ] window functions (frameless, resize, fullscreen, etc.) ✨
  - We may have some of these features or they may be in the lower level webview libs, but it might need to still be implemented in Tauri. We need to make sure this is documented and implement what is missing.

/cc @tauri-apps/core to add items, etc.

**Comment by sanket143** [comment]
> any starter projects for the big frameworks (react is the only one so far) sparkles

Can I add a starter project using Vue 3? Where can I make the contributions?

**Reply by nklayman** [reply]
Thanks for the offer, but we have a special vue cli plugin that works with Vue 3: https://github.com/tauri-apps/vue-cli-plugin-tauri. However, this will need to be integrated into `create-tauri-app`, and you may be able to help with that. Unfortunately, `create-tauri-app` is not ready yet, though I plan to finish it sometime this week if I can get all my other stuff done first. I'll let you know when I finish it if you want to help integrate it with the cli plugin.

**Reply by sanket143** [reply]
Is there any work-in-progress public repo for `create-tauri-app`?

**Comment by nklayman** [comment]
Hacktoberfest has been over for a while now, so I'm gonna mark this as answered.
